easybuild/easyconfigs/s/Siesta/Siesta-4.1-b2-intel-2017a.eb

```diff
@@ -0,0 +1,20 @@
+name = 'Siesta'
+version = '4.1-b2'
+
+homepage = 'http://departments.icmab.es/leem/siesta'
+description = """SIESTA is both a method and its computer program implementation, to perform efficient electronic
+ structure calculations and ab initio molecular dynamics simulations of molecules and solids."""
+
+toolchain = {'name': 'intel', 'version': '2017a'}
+toolchainopts = {'usempi': True}
+
+sources = [SOURCELOWER_TAR_GZ]
+vl = version.split('-')
+source_urls = ['https://launchpad.net/siesta/%s/%%(version)s/+download' % vl[0]]
+checksums = ['6dd700aa79b0da39bbd3233c220d630f']
+
+dependencies = [('netCDF-Fortran', '4.4.4')]
+
+parallel = 1
```

Comment:
versions newer than `4.1` support building in parallel (according to the easyblock)?